examples/keys/keys.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"github.com/BuxOrg/go-buxclient/utils"
+)
+
+func main() {
+	// Generate a random set of keys
+	hdXpriv, hdXpub := utils.GenerateRandomSetOfKeys()
+
+	fmt.Println("Your XPriv: ", hdXpriv)
+	fmt.Println("Your XPub: ", hdXpub)
+
+	xpub, err := utils.GetPublicKeyFromHDPrivateKey(hdXpriv.String())
```

Comment:
I like this method because I often had saved xpriv and did not have pub which I have to put into bux-server. This method allows me to use my key and in easy way get pub for it

Reply:
I agree with Paweł. For some reason I also have this situation quite often